java/distance-measurement-analysis/src/main/java/com/esri/arcgisruntime/sample/distancemeasurementanalysis/MainActivity.java

```diff
@@ -0,0 +1,217 @@
+import com.esri.arcgisruntime.geoanalysis.LocationDistanceMeasurement;
+import com.esri.arcgisruntime.geometry.Distance;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AnalysisOverlay;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.DefaultSceneViewOnTouchListener;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+
+public class MainActivity extends AppCompatActivity {
+
+  private SceneView mSceneView;
+  private TextView mDirectDistance;
+  private TextView mVerticalDistance;
+  private TextView mHorizontalDistance;
+
+  private LocationDistanceMeasurement distanceMeasurement;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    // create a scene and add a basemap to it
+    final ArcGISScene scene = new ArcGISScene();
+    scene.setBasemap(Basemap.createImagery());
+
+    // inflate views from layout
+    mSceneView = findViewById(R.id.sceneView);
+    mDirectDistance = findViewById(R.id.totalDistance);
+    mHorizontalDistance = findViewById(R.id.horizontalDistance);
+    mVerticalDistance = findViewById(R.id.verticalDistance);
+    Spinner mUnitSpinner = findViewById(R.id.units_spinner);
+    // set the scene to the view
+    mSceneView.setScene(scene);
+
+    // add base surface for elevation data
+    Surface surface = new Surface();
+    surface.getElevationSources().add(new ArcGISTiledElevationSource(getResources().getString(R.string.world_terrain_3d)));
+    surface.getElevationSources().add(new ArcGISTiledElevationSource(getResources().getString(R.string.MNT_IDF)));
+    scene.setBaseSurface(surface);
+
+    // add building layer
+    final String buildings = getResources().getString(R.string.buildings_tile);
+    ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(buildings);
+    scene.getOperationalLayers().add(sceneLayer);
+
+    // create analysis overlay and add it to scene
+    AnalysisOverlay analysisOverlay = new AnalysisOverlay();
+    mSceneView.getAnalysisOverlays().add(analysisOverlay);
+
+    // initialize a distance measurement and add it to the analysis overlay
+    Point start = new Point(-4.494677, 48.384472, 24.772694, SpatialReferences.getWgs84());
+    Point end = new Point(-4.495646, 48.384377, 58.501115, SpatialReferences.getWgs84());
+    distanceMeasurement = new LocationDistanceMeasurement(start, end);
+    analysisOverlay.getAnalyses().add(distanceMeasurement);
+
+    // zoom to initial measurement
+    Camera camera = new Camera(start, 375.0, 0.0, 45.0, 0.0);
```

Comment:
Can we change the camera to allow both start and end points to be in view when the app starts? Something like:

Camera camera = new Camera(start, 300.0, 315.0, 60.0, 0.0);